js/projection/pg-chain.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'underscore';
+
+class ProjectionChain extends Backbone.Model {
+			initialize(options) {
+				this.projections = [];
+				this.on('change', () => this.update());
+			}
+
+			update() {
+				return _.reduce(this.projections, 
+					(memo, projection) => projection(memo, this.attributes),
+					{}
+					);
+			}
+
+			pipe(projection) {
+				var self = this;
```

Comment:
not necessary, just use this. As you are writing in ES6, it's not necessary to assign `this` to `self`